src/components/mainContent.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { FeatureProductsData } from './FeatureProductsData';
+
+const MainContent = () => {
+    const listItems = FeatureProductsData.map((item) =>
```

Comment:
This is not only for this line but as a general advice, usually only the components are named with capital letters at the beginning, for example main content would be correct but FeatureProductsData would be better with no capital letter at the begining.